### PR TITLE
List, TagBox, Splitter: localize aria-roledescription attribute (T1267579)

### PR DESCRIPTION
**Pull request by anna-shakhova**

### CODE REVIEW

File: packages/devextreme/js/__internal/ui/splitter/resize_handle.ts [hunk 1]
```diff
@@ -305,7 +305,7 @@ class ResizeHandle extends Widget<ResizeHandleOptions> {
     this.setAria({
       role: 'application',
       // eslint-disable-next-line spellcheck/spell-checker
-      roledescription: 'separator',
+      roledescription: messageLocalization.format('dxSplitter-ariaRoleDescription'),
```
Comment (ksercs) [inline packages/devextreme/js/__internal/ui/splitter/resize_handle.ts]: While we're localizing `separator` not a "splitter" itself I think we should show it explicitly in the variable's name. Maybe smth like this?
```suggestion
      roledescription: messageLocalization.format('dxSplitter-separatorAriaRoleDescription'),
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js [hunk 2]
```diff
@@ -278,6 +278,9 @@ if(devices.real().deviceType === 'desktop') {
                         items: [{ text: 'Item_1' }, { text: 'Item_2' }, { text: 'Item_3' }],
                     }, options))
             });
+            const localizedRoleDescription = 'list';
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js]: If you trying to check that custom value can be loaded to the dictionary, let's use move explicit custom value
```suggestion
            const localizedRoleDescription = 'My Custom List';
```

File: packages/devextreme/js/__internal/ui/splitter/resize_handle.ts (hunk 1 above)
Comment (anna-shakhova) [reply packages/devextreme/js/__internal/ui/splitter/resize_handle.ts]: renamed

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js (hunk 2 above)
Comment (anna-shakhova) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js]: fixed

File: packages/devextreme/js/localization/messages/en.json [hunk 3]
```diff
@@ -372,6 +373,7 @@
         "dxTagBox-allSelected": "All selected ({0})",
         "dxTagBox-moreSelected": "{0} more",
         "dxTagBox-tagRoleDescription": "Tag. Press the delete button to remove this tag",
+        "dxTagBox-ariaRoleDescription": "Tagbox",
```
Comment (vladaskorohodova) [inline packages/devextreme/js/localization/messages/en.json]:
```suggestion
        "dxTagBox-ariaRoleDescription": "Tag box",
```

Comment (anna-shakhova) [reply packages/devextreme/js/localization/messages/en.json]: fixed

**Review by vladaskorohodova — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**